src/interface/gemm_interface.hpp

```diff
@@ -28,9 +28,10 @@
 #include "interface/blas1_interface.h"
 #include "interface/blas3/backend/backend.hpp"
 #include "interface/blas3_interface.h"
+#include "interface/extension_interface.h"
```

Comment:
Do we need this header here?

Reply:
Typo left from some experiments, will remove it!